python_modules/dagster/dagster/_utils/test/__init__.py

```diff
@@ -198,6 +220,56 @@ def execute_solids_within_pipeline(
     return {sr.solid.name: sr for sr in result.solid_result_list}
 
 
+def execute_op_for_test(
```

Comment:
It's a little strange to call a function like this `for_test`, because nothing about its implementation is test-specific.  Maybe `wrap_op_in_graph_and_execute`?

And a thought: if we changed this to be a function that accepts an op and returns a graph that wraps it, we could maybe expose it in the future as a useful utility.